README.md

```diff
@@ -52,7 +52,10 @@ the details of your repo to produce a setting in the form:
 
 ```ruby
 REPO = 'https://YOUR_GITHUB_TOKEN@github.com/everypolitician-scrapers/kenya-mzalendo'
-ScrapedPageArchive.github_repo_url = REPO
+storage = ScrapedPageArchive::GitStorage.new
+storage.github_repo_url = REPO
```

Comment:
> Constructors that don't take any arguments are a warning sign.

mindblown.gif

Very interesting observation! I hadn't really thought about it in those terms before but the more I think about it the more sense it makes. I've moved this option into the constructor in https://github.com/everypolitician/scraped_page_archive/pull/42/commits/23ecf52a861d1098358dde19e4fd65e5f6b93a5e.

> Do all the methods in these classes need to be public?

No, indeed they don't. I've defined a small public interface in https://github.com/everypolitician/scraped_page_archive/pull/42/commits/fb8f98d599f1752d19f52c1892590fe61065ddd6.